Student/Canvas/Shrug/LegacyModuleProgressShim.swift

```diff
@@ -17,15 +17,14 @@
 //
 
 import Foundation
-import TechDebt
 import CanvasKit
 import CanvasCore
 
-/** Keeps TechDebt Modules and Assignments UI up to date course progress
+/** Keeps Modules and Assignments UI up to date course progress
  */
 open class LegacyModuleProgressShim: NSObject {
     @objc public static func observeProgress(_ session: Session) {
-        NotificationCenter.default.addObserver(session, selector: #selector(Session.legacyModuleItemProgressUpdated(_:)), name: NSNotification.Name.CBIModuleItemProgressUpdated, object: nil)
+        NotificationCenter.default.addObserver(session, selector: #selector(Session.legacyModuleItemProgressUpdated(_:)), name: NSNotification.Name(rawValue: "CBIModuleItemProgressUpdated"), object: nil)
```

Comment:
FileDetailsViewController & others post this notification, but rather than create new constants, I figured I'd just use the literal strings, since there is likely some other cleanup we want to do for the module notification system.